### PR TITLE
feat: improve the readability of `circleci-env apply`

### LINKED ISSUE #28: Improve the readability of `circleci-env apply --dry-run`
## Problems to solve

Currently, it is hard to read the result of `circleci-env apply --dry-run` if many repositories are managed by this tool, because
the result of all repositories are outputted including the result of unchanged repositories.
And both `envvars` and `ssh_keys` are outputted even if only one of them is updated.

```
=== github/xxx/xxx

Progress(dry-run): |
envvars:
ssh_keys:
  no change

=== github/xxx/xxx

Progress(dry-run): |
envvars:
ssh_keys:
  no change

...

=== github/xxx/xxx

Progress(dry-run): |
envvars:
  + add    GITHUB_TOKEN=xxx
ssh_keys:
  no change

=== github/xxx/xxx

Progress(dry-run): |
envvars:
ssh_keys:
  no change

...

```

## Solution

By outputting only repositories which are updated, the result would be easy to read.

```
=== github/xxx/xxx

Progress(dry-run): |
envvars:
  + add    GITHUB_TOKEN=xxx
```

### PR DESCRIPTION
**Pull request by suzuki-shunsuke**
Close #28 

## Manual test

- [x] output only updated repositories
- [x] output only envvars

### CODE REVIEW

File: lib/circleci/env/command/apply.rb [hunk 1]
```diff
@@ -29,8 +29,15 @@ def run
           load_config(@config)
 
           puts "Apply #{@config} to CircleCI#{dry_run? ? ' (dry-run)' : ''}"
+          changed = false
           DSL::Project::projects.each do |proj|
-            apply(proj)
+            if !no_change?(proj) then
```
Comment (ujihisa) [inline lib/circleci/env/command/apply.rb:34]: [nits]
```suggestion
            unless no_change?(proj)
```

Comment (suzuki-shunsuke) [reply lib/circleci/env/command/apply.rb]: Thanks!

**Comment by suzuki-shunsuke** [comment]
https://github.com/quipper/circleci-env/issues/28#issuecomment-668891439

**Comment by suzuki-shunsuke** [comment]
I close this pull request due to https://github.com/quipper/circleci-env/issues/28#issuecomment-668891439 .

**Pull request closed without merging**